server/src/events.mjs

```diff
@@ -161,20 +256,34 @@ function sendBroadcastEvent(ws, userId, method, result, msg, fSuccess, fErr, fFa
   coreSendEvent(true, ws, userId, method, result, msg, fSuccess, fErr, fFatalErr);
 }
 
-// sending response to web-socket
+/**
+ * Emits a response to the registered event listener.
+ * @param {any} finalResult - The final result to be included in the response.
+ * @param {string} msg - The message associated with the response.
+ * @param {string} userId - The ID of the user.
+ * @param {string} method - The method associated with the event.
+ * @returns {void}
+*/
 function emitResponse(finalResult, msg, userId, method) {
   const listener = getRegisteredEventListener(userId, method);
   if (!listener) {
     logger.debug('Event message could not be sent because a listener was not found');
     return;
   }
-  const { id, wsArr } = listener;
-  const oEventMessage = {
-    jsonrpc: '2.0',
-    id: id,
-    result: finalResult
+
+  const { metadata, wsArr } = listener;
+  // Defines the data object that will be inputted into handlebars
+  const templateData = {
+    ...metadata,
+    result: finalResult,
+    resultAsJson: JSON.stringify(finalResult)
   };
-  const eventMessage = JSON.stringify(oEventMessage);
+
+  // If event template config does not exist, use default template which just returns the result
+  const defaultEventTemplate = `{"result":{{{resultAsJson}}}{{~"}}"~}}}`;
```

Comment:
We shouldn't do any formatting at all unless a template exists. Just send the exact event payload that was given to the API